packages/docs/blog/2023-09-06-graphql-vs-rest.md

```diff
@@ -0,0 +1,187 @@
+# GraphQL vs REST APIs in Medplum
+
+One of the most frequent questions we get from our users is whether they should use Medplum's [REST](http://hl7.org/fhir/R4/http.html) or [GraphQL](https://www.medplum.com/docs/graphql) APIs. Both have a FHIR specification, but they offer different tradeoffs for different use cases.
+
+In this post, we'll discuss these tradeoffs and provide some guidance on how you can choose which API is right for you.
+
+## GraphQL
+
+GraphQL has [surged in popularity](https://devops.com/key-findings-from-the-2022-state-of-graphql-report/#:~:text=GraphQL%2C%20the%20open%20source%20query,the%20specific%20client%20at%20hand.) in recent years.
+
+In the context of FHIR, one of GraphQL's strongest features is the ability to quickly retrieve [multiple linked resources](/docs/graphql/basic-queries#resolving-nested-resources-with-the-resource-element). While the REST API allows similar functionality using the [`_include`](/docs/search/includes) and [`_revinclude`](/docs/search/includes) search parameters, GraphQL offers a more natural syntax for querying bundles of resources that reference each other.
+
+```graphql
```

Comment:
Consider adding a link to the graphiql tool there to remind people that it exists.